contrib/bundle/test-e2e

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Check for root
+if [[ $EUID -ne 0 ]]; then
+    echo "Please run this script as root"
+    exit 1
+fi
+
+# Assume we're running on this arch
+ARCH=amd64
+
+# Global vars to be used
+# shellcheck source=vars
+source "$(dirname "${BASH_SOURCE[0]}")"/vars
+
+pushd "$ARCHIVE_PATH"
+
+# Untar the bundle
+tar xfvz "$ARCHIVE"
+pushd cri-o
+
+# Install and prepare config
+./install
+
+curl https://raw.githubusercontent.com/cri-o/cri-o/master/scripts/get | bash
```

Comment:
part of the intention of the issue was to catch issues before we merge. Unfortunately, this seems to only be able to catch them after. We do get a signal after the fact (tests start failing, we know which commit did it), but we don't prevent prow from going red upstream, which looks bad for the stability of the test.

I personally don't know how to inject the current binary somewhere, then have this pull it from there, but ideally that's what would happen here. does that seem reasonable?

Reply:
This line has to be removed, we pass the binary artifact during the build. Thanks for the hint!